### DISCUSSION TITLE
Evangelion Theme Draft (3.5" | Landscape)

### DISCUSSION THREAD

**Discussion by haidara02** [mathoudebine/turing-smart-screen-python#337] [Themes]
My first theme based on NGE countdown UI. Whipped this up quickly so I mainly used fonts from the release aside for one static text that says Evangelion (font is included but you can change it). Not sure why there is a notdef character in the clock, might need to raise an issue for that.
![image](https://github.com/mathoudebine/turing-smart-screen-python/assets/62708516/602f8047-29bd-4e62-8ed6-dbbee611602a)

Just unzip and paste contents inside res folder into your res folder
[EvaClassic.zip](https://github.com/mathoudebine/turing-smart-screen-python/files/12717417/EvaClassic.zip)

**Comment by mathoudebine** [comment]
Very nice work! Thanks for sharing with the community!
